test/unit/range/iterator_test_template.hpp

```diff
@@ -250,6 +251,7 @@ TYPED_TEST_P(iterator_fixture, move_forward_pre)
                                          std::ranges::end(this->test_range),
                                          this->expected_range);
 
+    // FIXME: a const_iterable input iterator would be iterated over a second time which is not allowed
```

Comment:
```suggestion
}

TYPED_TEST_P(iterator_fixture, move_forward_pre_const)
{
```
Should fix it because in every test the variables are fresh right?